lib/pi_piper/sysfs/sysfs.rb

```diff
@@ -0,0 +1,73 @@
+module PiPiper
+  class Sysfs < Driver
+
+    def initialize
+      @exported_pins = Set.new
+    end
+
+    def close
+      unexport_all
+      @exported_pins.empty? 
+    end
+    
+# Support GPIO pins
+    GPIO_HIGH = 1
+    GPIO_LOW  = 0
```

Comment:
I intended to  group resources by API section : pin, pwm, i2c, spi, ...
ad let general fonctionality (initialize, close, ...) at the top.